### DISCUSSION TITLE
Megane 3 BOSE - TPMS problem with new sensors

### DISCUSSION THREAD

**Discussion by LemurMort** [cedricp/ddt4all#854] [Q&A]
Hi all!
Recently I bought 4 new sensors from aliexpress and had them installed.
After a few miles they were not recognized and I got a red warning to check my sensors.
First, in dashboard I got to tpms menu, held a button for 5sec and got message that pressure ref is active or something like that but the error didn't dissapear after that.
Then, in DDT4ALL I went to BCM, TPMS extra screen and wrote IDs from those new sensoes. Then drove a couple of hundred meters and they were all read and pressure was shown on dashboard. Excellent, I thought.

The next day, I went home from work and the red message appeared again. No pressures showing, I hold the button for 5sec again and no results. In DDT the IDs I wrote are still there.

Anyone knows whats happening? 
Are batteries in sensors dead? 
Thanks!

EDIT: Also, in DDT4ALL I see there are two labels (jeu ete and jeu hiver) for summer and winter wheel set but I can only write sensor IDs in one label (see pic). Why is that? 
![IMG_20230612_182253](https://github.com/cedricp/ddt4all/assets/131789389/7f57f3b7-9fa2-4b46-95f1-d550bc3a7981)

**Comment by matzel82** [comment]
Sensor not compatible or not programt. 

Possible you have to clone the old sensors

**Reply by LemurMort** [reply]
If so, how was everything ok for 1 day after I manually wrote sensor IDs?

How to clone them?

**Reply by matzel82** [reply]
You either need original sensors that are already described or are looking for how you can clone from the original sensors.

NOT from Chinese store !!!!!!

**Comment by mcquaterx** [comment]
This is not an answer... But I have currently got the same problem, which is why I've just got DDT4ALL set up to investigate. I used 4 new sensors from AliExpress, and programmed the codes in the 2015 Megane 3 CC GT-Line using a Foxwell 680 Pro. 
They were fine for 100+ miles over 2 weeks. Then after stopping for an hour on a journey they ALL stopped being read by the car..they can't all have 'failed' at the same time. 3 different TPMS scanners say they are OK with good batteries and correct codes 
I have the old sensors, and triggered them in the car after reprogramming the car with their codes. The car still says no TPMS sensors...
I have reprogrammed the new sensors and activated them with an Autel (Otofix 608)TPMS programmer. It still reads them ok, but the car will not see any Sensors..  
If anyone has found an answer, I'd like to know. If I find the answer, I shall post it here ... 
If I can't find the answer , I shall try to remove TPMS using DDT4ALL, and use an external tpms for passing MOTs.

**Comment by AndyIreland** [comment]
in DDT4ALL if your cars ECU's have settings for 'Direct TPMS' and 'Indirect TPMS' try both - (but I think it should be indirect TPMS myself) one uses sensors in the valves the other uses sensors by ABS I think.

**Reply by matzel82** [reply]
That's correct

**Reply by mcquaterx** [reply]
I'm waiting on a new Battery for the car before I scan it to see what's going on, but mine is a late model Megane Mk3 . It looks like there are 6 or more console options for the ECU, so I'm hoping DDT4ALL will correctly identify mine or I'll be writing data in the wrong space. 
But I'm fairly sure that on the UCH setting, the choice will be for 'Without Reset' and 'With Reset' since MK3 PH2 can reset tyre pressure normal range .
I am hoping that the Foxwell Analyser that wrote the TPMS values mis identified the car as MK4 and was able to set the TPMS option to Indirect TPMS, which I'd guess would only take effect after a reboot... Because my battery was failing and the stop I made where the TPMS failed could have been the point where the voltage dropped and restarted that computer... (I have a BMW motorcycle which does that to the ABS system if the battery is anything but fully charged)
Thanks for the hints though...
Once my battery comes, I'll check out the theory...

**Reply by mcquaterx** [reply]
I got my new battery...old one tested at 66%... Put it in the car and started it...no warning lights, car started almost instantly... Left the car unused for a fortnight for various reasons and put the charger on for a day before taking out for the first time with the new battery... I was expecting it to run for 10-15: minutes and then realise it could not read TPMS sensors, so I hadn't even scrolled through to see if readings were there. 
So after 15 minutes I checked since no warnings had come up, and all 4: sensors were reading OK...
I'm glad that it is working, and fortunately I had been too wary about removing the TPMS with ddt4all ... but that would mean that whatever processor reads the TPMS does not function correctly with a severe voltage drop caused by startup on the diesel. However, one of the codes stored showed a defective glow plug relay connection, which also has not returned after clearing it...
I've only driven 40 miles so far ...if the fault returns in the next 1000 miles, I'll leave an update, but here's hoping...!

**Reply by Vetal101** [reply]
Hello! I have exactly the same problem. An error message appears: “Check the tire pressure sensor.” The pressure is normal (2.2-2.3bar). The wiper switch button does not reset this error. You have to remove the battery terminal to clear this error. There is no error for several days and I can drive 100-200 km without an error. Then the error pops up again. And again you have to remove the battery terminal.
Tell me how you solved this problem. I didn't understand something. Have you changed the battery?!
![photo_2024-08-20_21-16-21 (2)](https://github.com/user-attachments/assets/914246b2-3afd-48e0-8821-c41945e87baa)
![photo_2024-08-20_21-16-21](https://github.com/user-attachments/assets/a1530977-c718-434f-a54c-9011249a74df)

**Reply by mcquaterx** [reply]
I'm still working on a permanent solution...  my temporary one was to install a remote battery isolator which was £20 on AliExpress or £25 on eBay... If I disconnect for 25 minutes (disconnect after locking but before alarm arms) then I get 12  restarts of the engine...after the last start you get about a kilometre and although all 4 tyre pressures show OK, they all then disappear... NOT a function of Distance, purely the number of starts. 
I have new batteries to install on the original TPMS valves and will replace one at a time to see if just the new ones 'disappear' , or if all or none continue to vanish.
All the dealers and specialists deny all knowledge of any cause or cure... Renault garages simply blame using a programmer that isn't a CLIP... I've tried 3 so far, including a FoxPro professional and an Otofix (same as an Autel Maxipro 908S).
I will post again once I find the cure...

**Reply by ottomanjjj** [reply]
Hello;

Years have passed but I wanted to write about the problem I experienced. I wanted to add this feature that was not originally in my vehicle but had an infrastructure. 3 of the sensors I received from my part are different brands, 1 is original VDO. Since the part is familiar, I bought it on the condition of returning it. I introduced it with dtt4all without installing it on the tires, but since it was not installed, after a while I received a flat tire warning and after a while I received a warning that the tire sensor could not be detected and the tire symbols on the dial disappeared. We thought this was normal and installed the sensor during the winter tire change yesterday. However, when I wanted to introduce it again in the evening, I could only introduce the VDO one. When I find the opportunity, I will repeat the processes I did from the beginning and give information here.

The introduction process was completed after a long effort. In fact, a nail got into one tire and I understood it thanks to the sensor. The tire repaired it but I think he pressed 40 psi after the repair and 3 sensors except VDO could not be detected. When I connected with PyREN, I saw that VDO was "status 1" and the others were "status 2".  The pressure was showing as 3.5 bar. I lowered the pressure and when I increased it to about 20 psi, it suddenly detected the sensor and gave a flat tire warning. However, when I pressurized it to 30 psi, it still could not detect the sensors. I could not understand what to do.

**Comment by Marianpol** [comment]
For those who have problem with disappearing TPMS sensor reading after few days I have recommendation to change  PARA_LOC_LEARNING_DEFAULT from 10 to 255. This will extend time left to next battery disconnection to around 5 months.

**Reply by ottomanjjj** [reply]
Thank you very much for your support.

**Reply by mcquaterx** [reply]
I used ddt4all to do as you suggested. It was good for 7 months, and still allowed me to disconnect the battery after this and I still have no multiple sensors disappearing after another couple of months. 
I did get a sensor missing message after putting my spare wheel on after a puncture on Front Right (it has the original Renault sensor), and when I drove off. 
It appears that the Renault Sensor overwrote itself without having to reprogrammed manually, but it coded the wrong positionie Front Left although pressure reading on the dash was in the correct location ie Front Right, but this meant the sensor in the good un-punctured wheel was ignored, hence the 'sensor missing ' message. 
I have 2 good OBD2 analysers , neither of which would then read any sensor codes, but ddt4all accessed these and allowed me to rewrite the TPMS codes and fix that issue.

**Reply by Renotech1** [reply]
Just disable the TPMS and no problem 🤷‍♂️
I you want to get it work. But good sensors and learn them🧐

> I used ddt4all to do as you suggested. It was good for 7 months, and still allowed me to disconnect the battery after this and I still have no multiple sensors disappearing after another couple of months. 
> I did get a sensor missing message after putting my spare wheel on after a puncture on Front Right (it has the original Renault sensor), and when I drove off. 
> It appears that the Renault Sensor overwrote itself without having to reprogrammed manually, but it coded the wrong positionie Front Left although pressure reading on the dash was in the correct location ie Front Right, but this meant the sensor in the good un-punctured wheel was ignored, hence the 'sensor missing ' message. 
> I have 2 good OBD2 analysers , neither of which would then read any sensor codes, but ddt4all accessed these and allowed me to rewrite the TPMS codes and fix that issue.

**Reply by mcquaterx** [reply]
Disabling the TPMS isn't a viable option since I can't identify which display of the three options apply to my car. Its a GT line TomTom 2015 CC and I can't tell which options apply...it looks like there's 3 stages to removing the TPMS, and I don't want to end up screwing up the BCM.
If anyone knows exactly how , feel free to enlighten me...

**Reply by Renotech1** [reply]
Probely you doing something wrong. As its an older car i am 100% sure you can disable the tpms 🫡
> Disabling the TPMS isn't a viable option since I can't identify which display of the three options apply to my car. Its a GT line TomTom 2015 CC and I can't tell which options apply...it looks like there's 3 stages to removing the TPMS, and I don't want to end up screwing up the BCM.
> If anyone knows exactly how , feel free to enlighten me...

**Reply by Renotech1** [reply]
Look for TPMS / SSPP in cluster/tdb do disable the tpms function

![Screenshot_20251214_184658_Ecu Tweaker.jpg](https://github.com/user-attachments/assets/7bd2b837-b268-4699-99bb-09953c9ec119)

![Screenshot_20251214_184723_Ecu Tweaker.jpg](https://github.com/user-attachments/assets/59d94374-086d-4345-86b9-72bf51349218)

> Probely you doing something wrong. As its an older car i am 100% sure you can disable the tpms 🫡
> > Disabling the TPMS isn't a viable option since I can't identify which display of the three options apply to my car. Its a GT line TomTom 2015 CC and I can't tell which options apply...it looks like there's 3 stages to removing the TPMS, and I don't want to end up screwing up the BCM.
> > If anyone knows exactly how , feel free to enlighten me... 
> 
>

**Reply by mcquaterx** [reply]
At the risk of appearing rude...no I'm not doing anything wrong.. disabling the TPMS on this car requires more than one action...if you can't provide useful information, then please do not comment, but if you know exactly what to do then feel free to leave appropriate information...thanks..

**Reply by Renotech1** [reply]
Hmmm oke her 18y specialist.
Did tmps off on megan 3 this week..
Figure it out yourself ... just trying to help 🤷‍♂️🤷‍♂️
> At the risk of appearing rude...no I'm not doing anything wrong.. disabling the TPMS on this car requires more than one action...if you can't provide useful information, then please do not comment, but if you know exactly what to do then feel free to leave appropriate information...thanks..

**Reply by mcquaterx** [reply]
To Renotech1...
If you have taken the time to make a comment on how easy it is and say that I probably don't know what I'm doing, and claim you know exactly how to disable the TPMS but can't be bothered to give the slightest explanation or instructions then I don't know what you're doing on here...I thought the idea of GitHub was to support and help other users...
Obviously I don't know what I'm doing with ddt4all, because if I did I wouldn't be on here looking for answers for myself, I'd be offering useful instructions and information to others.
I'll look forward to a step by step instruction to disable my TPMS and leave no warning lights or trace that it's disabled...

**Reply by Renotech1** [reply]
I just trying to help bro.. I just say what you try to do .. in tdb to disable... i even send you screenshot..
I dont say you dont have knowledge or so.
I here to help. Like nobody else answered this 🧐. If you dont like my help dont answer me🙏


>  To Renotech1...
> If you have taken the time to make a comment on how easy it is and say that I probably don't know what I'm doing, and claim you know exactly how to disable the TPMS but can't be bothered to give the slightest explanation or instructions then I don't know what you're doing on here...I thought the idea of GitHub was to support and help other users...
> Obviously I don't know what I'm doing with ddt4all, because if I did I wouldn't be on here looking for answers for myself, I'd be offering useful instructions and information to others.
> I'll look forward to a step by step instruction to disable my TPMS and leave no warning lights or trace that it's disabled...

**Reply by LemurMort** [reply]
I don't remember correctly but if it can help you...
You have to disable the TPMS system aaaand the TPMS display.
So, somewhere inside BCM you disable the system and then in Dashboard you disable display of messages and also TPMS menu.
I did it and then activated it again some 3 years ago so apologies for lack of details. But I certainly did it in those two mentioned modules.

**Reply by hnizil** [reply]
> For those who have problem with disappearing TPMS sensor reading after few days I have recommendation to change PARA_LOC_LEARNING_DEFAULT from 10 to 255. This will extend time left to next battery disconnection to around 5 months.

thanks , it solved my problem

**Comment by ottomanjjj** [comment]
I had disconnected the battery cable yesterday. When I connected it today, the left front sensor code that I had entered incorrectly before appeared. When I corrected the left front sensor code, all sensors worked without any problems. If the same thing happens again, I will enter all sensor codes as "000000", disconnect the battery cable and then enter them correctly. I also increased the PARA_LOC_LEARNING_DEFAULT value from 10 to 255.

**Comment by LemurMort** [comment]
Hello all!
Just to add my experience from the previous 2 years.
Each time I put the set of wheels that has chinese sensors I write their IDs manually.
I have also increased the mentioned parameter to 255 and everything works OK for around 5 months. After that, I have to disconnect battery for a few minutes, reconnect and then it again works for the next month or so. Then I have to replace the wheels again (with ones that have original sensors) and they are recognized automatically.

So, every year it's the same. 
It's not a big deal for me to find a few minutes and program that stuff.

Thank you! 🙂

**Comment by AndyIreland** [comment]
when I had my Clio4 I had to change the presence spp to 0  in the BCM/uch ECU as well as turn it off on the dash 

<img width="1095" height="619" alt="image" src="https://github.com/user-attachments/assets/626d951a-ba8f-492a-920a-1295d605a88a" />

Dont forget when you change the setting in your cluster you have to also press on prise en complet button afterwards.

<img width="1260" height="609" alt="image" src="https://github.com/user-attachments/assets/cf4e4ddd-b7fc-4685-8947-938c6ad1a325" />




Good luck

**Reply by mcquaterx** [reply]
That highlights my problem with turning it off...I can't identify which dash I should select.
And before I search the forum to see if there's any info out there about ddt4all and the BMS...does anyone know if ddt4all can change the voltage control of the Battery Management System...my car swings from 11.8 to 15.1 volts and this ruins batteries, but worse than that , it means LCD headlight bulbs dim when you decelerate ( their optimal voltage is about 12.2 to 14.2 volts and I've tested at least a dozen different ones )...
Any takers for info on the BCM..? ( I've disconnected the voltage monitor chip on the -ve terminal to disable it for the moment)

**Reply by AndyIreland** [reply]
are you searching for the dash/cluster with the magnifying glass 
<img width="82" height="51" alt="image" src="https://github.com/user-attachments/assets/db82ba4c-e32d-4c59-9725-49d194d7845b" />
  in DDT4ALL?

**Comment by kostashel** [comment]
Hello everyone, Sorry for my bad English . I have had the same problem with the pressure sensors for two years. After visiting two official Renault dealerships and a local workshop, everyone told me that I had to replace the BCM at a cost of about 700 euros. At the same time as the sensor problem, I had a problem with the range of the wireless keys. After searching, I found that the pressure sensors and the keys operate at the same frequency of 433 MHz. I accidentally found that there were two cut wires at the point shown in the picture. After restoring the cut wires and resetting the BCM, the problem did not appear again and the problem with the keys stopped existing.
![52912679_x_b](https://github.com/user-attachments/assets/bd0b5c2b-24f5-43e4-bb9f-7d3a0a9d6cd6)
